app/modules/handler/containers/Stage/Identity.js

```diff
@@ -111,54 +99,34 @@ class PromptStageIdentity extends Component<Props> {
                     onSelect={onSelect}
                   />
                 </Form.Field>
-                <Form.Field>
-                  <Segment basic textAlign="center">
-                    <Header
-                      content="Options"
-                      size="small"
-                    />
-                    <Form.Checkbox
-                      checked={enableSessions}
-                      label="Create a session for future use with this app"
-                      onChange={this.props.toggleSessions}
```

Comment:
This I believe is a persistent object, which is saved in their settings. If we remove the checkbox and some users have it set to false, the issue with Wharf will never be resolved for them. 

We need to find the code that reads whether that value is on/off, and force it to always be true.

Reply:
Ah ok, good point 🤔

Reply:
Done 👍